doc/source/whatsnew/v0.25.0.rst

```diff
@@ -22,8 +22,8 @@ Other Enhancements
 - Indexing of ``DataFrame`` and ``Series`` now accepts zerodim ``np.ndarray`` (:issue:`24919`)
 - :meth:`Timestamp.replace` now supports the ``fold`` argument to disambiguate DST transition times (:issue:`25017`)
 - :meth:`DataFrame.at_time` and :meth:`Series.at_time` now support :meth:`datetime.time` objects with timezones (:issue:`24043`)
+- Add ``casefold`` to ``Series.str`` (:issue:`25405`)
```

Comment:
`Series.str` has gained the ``:meth:`Series.str.casefold` `` method to ......(fill this in)

Reply:
changed